site/_posts/2019-09-01-announcing-gh-move.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Announcing a new GitHub home for Velero
+image: /img/posts/vmware-tanzu.png
+excerpt: The next Velero release (v1.2) will be built out of a new GH organization, and we have significant changes to our plugins.
+author_name: Carlisia Campos
+author_avatar: /img/contributors/carlisia-campos.png
+categories: ['velero']
+# Tag should match author to drive author pages
+tags: ['Carlisia Campos']
+---
+
+## VMware Tanzu
+
+Back in November 2018, the [VMWare acquisition of Heptio][1] made the news and in December we, the Heptio teams, started on our journey of integrating into their fold. Back then, our Velero project was named Heptio Ark and, in February, our v0.11 release brought about [our new name][2].
+
+Nine months later, we come full circle with our latest and very exciting change: we are now part of a brand new GitHub organization: [VMware Tanzu][3]. VMware Tanzu is a new family of products and services for the cloud native world. With the Velero project being a cloud native technology that extends Kubernetes and is a member of the CNCF (Cloud Native Computing Foundation), it is only natural that it would be moved to sit alongside all of the VMware supported cloud native repositories. You can read more about this organization wide change in this [VMware blog post][4].
+
+## The new Velero
+
+The new Velero repository now can be found at https://github.com/vmware-tanzu/velero.
+
+Items to note:</br>
+* Past issues, PRs, commits, contributors, etc all have been moved to this repo</br>
+* The Velero images will continue to be hosted at https://gcr.io/heptio-images
+
+The next Velero release, version 1.2, will be built out of this new repository and is slated to come out at the end of October. The main [set of changes][5] for v1.2 is the restructuring around how we will be handling **all** `object store` and `volume snapshotter` plugins. Previously, Velero natively supported both types of plugins for AWS, Azure, and GCP (Google Cloud Platform). Going forward as of v1.2, this will no longer be the case.
```

Comment:
> Previously, Velero natively supported both types of plugins for AWS, Azure, and GCP (Google Cloud Platform). Going forward as of v1.2, this will no longer be the case.

This reads a little strongly to me and could lead to confusion. I think we want to convey that previously these plugins were in-tree (i.e. in the main Velero repo) and going forward they'll be out-of-tree and installed just like any other plugin, but they're still 100% supported and there's just going to be a small change to the install experience for users of them.